cmd/microcloudd/main.go

```diff
@@ -18,6 +18,8 @@ import (
 	"github.com/canonical/microcloud/microcloud/version"
 )
 
+const MinimumHeartbeatInterval = time.Millisecond * 200
```

Comment:
Exported declarations should have a comment. (static analysis should have caught this, guess it's incomplete in microcloud).